packages/app-mobile/components/screens/ConfigScreen/plugins/PluginBox.tsx

```diff
@@ -100,10 +100,35 @@ const PluginBox: React.FC<Props> = props => {
 	};
 
 	const renderRecommendedChip = () => {
+		const onRecommendedPress = () => {
+			Alert.alert(
+				'Recommended Plugin',
+				'The Joplin team has vetted this plugin and it meets our standards for security and performance.',
```

Comment:
```suggestion
				_('Recommended Plugin'),
				_('The Joplin team has vetted this plugin and it meets our standards for security and performance.'),
```

Consider making this text localizable.

Reply:
thanks for pointing it out.

Reply:
should I do it for the alert buttons too? namely, 'learn more' and 'ok'?